src/main/java/com/nike/cerberus/error/DefaultApiError.java

```diff
@@ -293,6 +293,11 @@
      */
     FACTOR_VALIDATE_FAILED(99243, "Failed to validate factor. Please try again or try a different factor.", SC_UNAUTHORIZED),
 
+    /**
+     * Call to trigger sms or call challenge for OneLogin is not implemented.
+     */
+    TRIGGER_CHALLENGE_NOT_IMPLEMENTED(99244, "Call to trigger sms or call challenge for OneLogin is not implemented.", SC_METHOD_NOT_ALLOWED),
```

Comment:
This should be 501 status code, please refer to https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html when choosing status codes.

Reply:
405 Method Not Allowed
```The method specified in the Request-Line is not allowed for the resource identified by the Request-URI. The response MUST include an Allow header containing a list of valid methods for the requested resource.```

vs

501 Not Implemented
```The server does not support the functionality required to fulfill the request.```